website/docs/docs/deploy/continuous-integration.md

```diff
@@ -50,3 +50,6 @@ When you push a new commit to a PR, dbt Cloud enqueues a new CI run for the late
 
 <Lightbox src="/img/docs/dbt-cloud/using-dbt-cloud/example-smart-cancel-job.png" width="70%" title="Example of an automatically canceled run"/>
 
+### Run slot treatment
+
+CI runs for accounts on the [Enterprise, Team, and Developer plans](https://www.getdbt.com/pricing) don't consume run slots so a CI check will never block a production run. 
```

Comment:
per our convo, i added this section back and added the bit about Enterprise, Team, Dev. i went with "Dev" instead of "Free" (per your sugg) to be consistent/align with the pricing page link. hope that works too!

Reply:
thanks for adding this back in! i see what you mean about the pricing page; however, in the backend, `developer` and `free` are two different plan types; free plans get the same treatment as enterprise (including CI not using run slots), but developer plans _don't_ get this feature

Reply:
oh i see, interesting. thanks for flagging.

is "free" referring to the 14-day trial?

Reply:
i'm not sure! i thought they were separate things, but looking at the pricing page is confusing me 😅

Reply:
in the backend, we offer `free` plans to our partners (which have enterprise features). this is how we categorize it in the backend. the developer plan is our forever-free plan as described on our pricing page.

Reply:
@knartovich just to clarify the below CI not using run slots don't apply to dev plans? meaning, CI will use up their run slot?

> free plans get the same treatment as enterprise (including CI not using run slots), but developer plans don't get this feature

Reply:
@mirnawong1 thanks for clearing that up!

> are you saying the feature of "CI runs not using run slots" don't apply to developer plans? meaning, CI will use up their run slots?

yes, this is correct! for enterprise, team, and free plans, CI runs don't count against an account's run slots; for dev plans, they do

Reply:
ah ok thank you for clarifying! @nghi-ly suggested adding this clarifying [here](https://github.com/dbt-labs/docs.getdbt.com/pull/4706#discussion_r1457577530)